python/mxnet/io.py

```diff
@@ -487,32 +487,26 @@ class NDArrayIter(DataIter):
     >>> labels = np.ones([10, 1])
     >>> dataiter = mx.io.NDArrayIter(data, labels, 3, True, last_batch_handle='discard')
     >>> for batch in dataiter:
-    ...    print batch.data[0].asnumpy()
-    ...    batch.data[0].shape
+    >>>         print batch.data[0].asnumpy()
+    >>>         batch.data[0].shape
     [[[ 36.  37.]
       [ 38.  39.]]
-
      [[ 16.  17.]
       [ 18.  19.]]
-
      [[ 12.  13.]
       [ 14.  15.]]]
     (3L, 2L, 2L)
     [[[ 32.  33.]
       [ 34.  35.]]
-
      [[  4.   5.]
       [  6.   7.]]
-
      [[ 24.  25.]
       [ 26.  27.]]]
     (3L, 2L, 2L)
     [[[  8.   9.]
       [ 10.  11.]]
-
      [[ 20.  21.]
       [ 22.  23.]]
-
      [[ 28.  29.]
```

Comment:
not sure if you want to remove the empty line. it may fail the future docstring test